app/controllers/api/v0/markets_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Api::V0::MarketsController < ApplicationController
+  def index
+    render json: Market.all
+  end
+
+  def show
+    render json: Market.find(params[:id])
```

Comment:
I will plan on refactoring these to have serialization. If any questions arise send me a dm on slack.